op-guide/migration.md

```diff
@@ -201,6 +198,29 @@ mysql> select * from t2;
 +----+------+
 ```
 
+### 3. Best practice
+In order to migrate data quickly (especially huge amount of data), you can refer to the following recommendations.
+
+- make dump data file as small as possible, it is best not to exceed 64M (parameter -F 64)
+- loader's parameter -t can be evaluated according to the number and load of tikv instances. For example, there are three tikv instances, -t can be set to 3 * (1 ~ n); while load of tikv is too high and the loader's log appears a large number of `backoffer.maxSleep 15000ms is exceeded`, it's better to decrease -t ; otherwise increase it.
+
+### A sample, and related configuration
+
+ - the total amount of dump files are 214G, single table with 8 columns, 2 billion rows
```

Comment:
The total size of the exported files is 214G. A single table has 8 columns and 2 billion rows.